docs/dietpi_tools.md

```diff
@@ -426,11 +426,63 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
 
 === "DietPi survey"
 
-    This option will upload information on a anonymous base. The aggregated results of all uploads can be seen on the [DietPi survey statistics website](https://dietpi.com/survey/).  
-    Run `dietpi-survey`.
+    #### What is DietPi Survey?
+
+    DietPi Survey allows the DietPi project to obtain general information regarding your system and installed software.
 
     ![DietPi-Survey screenshot](assets/images/dietpi-survey.jpg)
 
+    - No private data is sent. No one can identify you. No IP address is obtained.
+    - This data allows the DietPi project team to focus and improve areas based on popularity. Ensuring the most common devices and software titles will receive the most support and improvements.
+    - By opting out, you are potentially preventing DietPi from achieving the best future possible experience for everyone.
```

Comment:
I think it would be good to to use "opt out" here, since in fact the survey is an "opt-in" system, so by default no data is sent, only when the user interactively decides to do so. Also in the script, we use both, have an opt-in and opt-out button, but actually that may cause confusion 😅.

Another point. By not opting in, especially the own hardware and software usage is not recognised, so back to the hard decisions above, opting in helps to assure that we keep up support for software and hardware that you use.

Reply:
@MichaIng:  
Changed it to positive description: 

*Basically DietPi survey is an ***opt OUT*** system. By selecting ***opt IN***, you are helping us achieving the best future possible experience for everyone and hat we keep up support for software and hardware that you use.*

Please review.

Reply:
Ah, äh actually it's the wrong way round now 😄. DietPi is an "Opt-in" system, which means that you need to opt in to have data sent.

So also below:
```
By staying ***opt IN***, you are supporting...
```
=>
```
By selecting ***opt IN***, you are supporting ... by running the following command and follow the instructions:
```
and
```
You can ***opt OUT*** of DietPi Survey by running the following command and follow the instructions:
```
=>
```
You can always revert by selecting ***opt OUT*** to stop send data, which will as well clear the data file that you may have uploaded already.
```
or something like this.

Reply:
@MichaIng: Ok, next try.

I moved text to an admonition at the top of the DietPi Survey chapter:

`-----<snip>-----`
![grafik](https://user-images.githubusercontent.com/57687516/107129643-167cf280-68c7-11eb-9029-cfef0e0be7b1.png)
`-----<snip>-----`

I try to keep track that I also change the related comment within `dietpi.txt` from phpBB to the docs page.